services/ux-backend/handlers/onboardingtokens/handler.go

```diff
@@ -5,12 +5,12 @@ import (
 	"net/http"
 
 	"github.com/red-hat-storage/ocs-operator/v4/controllers/util"
-	"github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handlers"
 	"k8s.io/klog/v2"
 )
 
 const (
 	onboardingPrivateKeyFilePath = "/etc/private-key/key"
+	ContentTypeTextPlain         = "text/plain"
```

Comment:
The moving of the content type into a common handlers package made sense even without the new handler. Please do not undo it.

Reply:
done.